src/main/java/com/google/sps/data/User.java

```diff
@@ -5,12 +5,19 @@
 
 public final class User {
 
-  private final String name;
-  private final Collection<String> productArea;
+  private long id;
+  private String profileType;
```

Comment:
You could use an enum for this so that there are only two options (student and mentor): http://tutorials.jenkov.com/java/enums.html#:~:text=A%20Java%20Enum%20is%20a,were%20added%20in%20Java%205.

Reply:
Done